### PR TITLE
A11y-Quiz: default landmark complementary counterpart

### PR DESCRIPTION
**Pull request by YuriDevAT**
## Summary of changes

For the a11y quiz I created a question regarding the counterpart of the default landmark complementary when using semantic HTML.

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/accessibility-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,16 @@
 const accessibilityQuiz = [
+  {
+    Question:
+      "What is the semantic HTML counterpart to the default landmark complementary?",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:4]:
```suggestion
      "Which of the following HTML elements uses the default landmark complementary?",
```

File: src/data/accessibility-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+    Answer: "aside",
+    Distractor1: "contentinfo",
+    Distractor2: "section",
+    Distractor3:
+      "This default landmark does not exist.",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:9]:
```suggestion
    Distractor3: "div",
```

File: src/data/accessibility-quiz.ts [hunk 3]
```diff
@@ -1,4 +1,16 @@
...
+    Explanation:
+      "If using semantic HTML, use the HTML tag <aside> element to define a complementary landmark.",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:11]:
```suggestion
      "The semantic <aside> element uses the complementary landmark which is a supporting section of the document created to complement the main content.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by YuriDevAT** [comment]
Thanks for taking time to improve my written English 💟

**Pull request merged**